includes/polling/ports.inc.php

```diff
@@ -744,10 +744,14 @@
         $fields['ifOutErrors_rate'] = $port['ifOutErrors_rate'];
         $fields['ifInOctets_rate'] = $port['ifInOctets_rate'];
         $fields['ifOutOctets_rate'] = $port['ifOutOctets_rate'];
+        $fields['ifInBits'] = $port['stats']['ifInBits_rate'];
```

Comment:
Are you 100% sure this isn't possible? We'd prefer not to send calculated metrics across as this is something that the front end should deal with.